docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-extended.md

```diff
@@ -14,6 +14,8 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Private API is extended. The page describes why entities on the project side have to avoid private api extending. The project has to follow the rule to be able to correctly receive Spryker autoupdates.
```

Comment:
Maybe it's better to make "Private API" clickable and put the link on page that describes what private API is.

Reply:
++ 
Added link to /docs/scos/dev/architecture/module-api/declaration-of-module-apis-public-and-private.html